module/motion/ScriptEngine/data/scripts/docker/Stand.yaml

```diff
@@ -1,74 +1,74 @@
-- duration: 1000
```

Comment:
I am wondering why we have different scripts for `docker` and `webots` at all... am I missing something?

Reply:
The script used at runtime is based on the platform. `./b run ...` which we use to test runs in docker, but the submitted Robocup image uses the webots scripts.

Reply:
`docker` scripts could be used for `HardwareSimulator`, `webots` scripts are intended to be used with `Webots`

Reply:
Yeah, but until we find a way to make `./b run webotsrobocup` use `webots` instead of `docker`, duplicating the script is necessary for testing.